iroh-blobs/src/downloader/test.rs

```diff
@@ -10,7 +10,7 @@ use iroh_net::key::SecretKey;
 
 use crate::{
     get::{db::BlobId, progress::TransferState},
-    util::progress::{FlumeProgressSender, IdGenerator, ProgressSender},
```

Comment:
A bunch of these things were marked as unused imports for me.
I think they're not actually unused, because the `mod tests { use super::*; ... }` below actually use them, but I think it's not a bad idea to move these imports into the test module.

Reply:
maybe you used a different rust version, surprised this wasn't shown in CI before